api/envoy/extensions/filters/network/redis_proxy/v3/redis_proxy.proto

```diff
@@ -191,6 +214,103 @@ message RedisProxy {
     Route catch_all_route = 4;
   }
 
+  // When hash_slot_routes is configured, redis proxy will forward requests to upstream cluster
```

Comment:
@mattklein123 adding some comments to explain this config. Please share your thoughts!